sw/device/tests/dif/dif_rv_timer_sanitytest.c

```diff
@@ -75,6 +74,12 @@ bool test_main(void) {
                                 kDifRvTimerEnabled) == kDifRvTimerOk);
 
   uint64_t current_time;
+  // Logs over UART incur a large runtime overhead. To accommodate that, the
+  // timer deadline needs to be large as well. In DV simulations, logs are not
+  // sent over UART, so we can reduce the runtime / sim time with a much shorter
+  // deadline (30 ms vs 100 us).
+  uint64_t kDeadline =
+      (kDeviceType == kDeviceSimDV) ? 100 /* 100 us */ : 30000 /* 30 ms */;
```

Comment:
this LGTM, but it would be good to get @mcy opinion on whether he envisioned the `deviceType` enums were meant to be used this way.  I feel like it's good though.

Reply:
This is fine :)

Reply:
IMO this is precisely the way `kDeviceType` was envisioned to be used. The only requirement is for it to completely stay out of DIFs and the mask ROM. Correct me if I am wrong :)

Reply:
@mcy, ping!